typo3conf/LocalConfiguration.php

```diff
@@ -61,11 +61,11 @@
         'processor_path_lzw' => '/usr/bin/',
     ],
     'MAIL' => [
-        'transport' => 'sendmail',
+        'transport' => 'smtp',
         'transport_sendmail_command' => '/usr/sbin/sendmail -t -i ',
         'transport_smtp_encrypt' => '',
         'transport_smtp_password' => '',
-        'transport_smtp_server' => '',
+        'transport_smtp_server' => 'mailhog:1025',
```

Comment:
These changes should go into typo3conf/AdditionalConfiguration.php instead, and only for "Development/Docker" application context.